docs/userGuide/siteConfiguration.md

```diff
@@ -118,4 +118,8 @@ The ignore pattern follows the [glob pattern used in .gitignore](https://git-scm
 
 **The level of headings to be indexed for searching.** Default: `3` %%i.e., only headings of levels 1,2,3 will be indexed for searching%%.
 
+#### **`enableSearch`**
+
+**Specifies that the website should use MarkBind's search functionality.** Default: `true`. See [User Guide: Making the Site Searchable](userguide/makingthesitesearchable) for more details.

```

Comment:
Link is broken, should be `[User Guide: Making the Site Searchable]({{ baseUrl }}/userGuide/makingTheSiteSearchable.html)`.